lua/nvim-tree.lua

```diff
@@ -337,6 +338,21 @@ local function setup_autocommands(opts)
       end,
     })
   end
+
+  -- TODO #1545 remove similar check from view.resize
+  if vim.fn.has "nvim-0.9" == 1 then
+    create_nvim_tree_autocmd("WinResized", {
+      callback = function()
+        if vim.v.event and vim.v.event.windows then
```

Comment:
Not sure if it makes any difference but fallback receives an event afair.

Reply:
Do you mean the fallback in view.lua?

Sending multiple events would be undesirable...

Reply:
> Do you mean the fallback in view.lua?
> 
> 
> 
> Sending multiple events would be undesirable...

No, meant that you should be able to do 

```lua
callback = function(event)
```

Reply:
Of course, I'll give that a go.

Reply:
Unfortunately no. The `v:event` data is still not passed to the callback. I hope they add that at some point...